README.md

```diff
@@ -7,10 +7,6 @@
 * Unassigned (reserved for future assignment)
 * A noncharacter
 
-## License
-
-The use of the *Last Resort* font is covered by the terms of the [SIL Open Font License, Version 1.1](https://scripts.sil.org/OFL) license. 
-
```

Comment:
@kenlunde @annebright if we leave these lines in, then my change is only adding lines and so shouldn't conflict.